ipatests/test_integration/test_dns_locations.py

```diff
@@ -534,6 +534,10 @@ def test_ipa_ca_records(self):
 
         expected_servers = (self.master.ip, self.replicas[1].ip)
 
+        for host in (self.replicas[0], self.replicas[1]):
```

Comment:
The records are deleted and re-added with the method delete_update_system_records, which performs the operations on the master. It means that the test needs to wait for replication to finish **from the master** and the right way to do it is to use `tasks.wait_for_replication(master.ldap_connect())`. The method must be applied on the source of updates, not on the destination.
Since the replication topology is a star topology, replicas[0] and replicas[1] both have an agreement with the master and the call to `tasks.wait_for_replication(master.ldap_connect())` will return only when both replicas have received the update.
The code should read:
```
        ldap = self.master.connect()
        tasks.wait_for_replication(ldap)
```

Reply:
I'm sorry for late. Thanks for your comment. I will update it (maybe tomorrow).

Reply:
updated to follow the suggestion. Please review it.